### ISSUE TITLE
Lock esri-loader version

### ISSUE THREAD

**Issue by vvscode** [Esri/react-arcgis#75]
Currently, package has the non-strict version of `esri-loader`

https://github.com/Esri/react-arcgis/blob/master/package.json#L9

so it makes work unpredictable. Could we have a frozen version?

**Comment by jgravois** [comment]
not locking the version of `esri-loader` allows us to make the latest release of the JSAPI available to developers without having to to tag a release in this project.

if you're interested in a specific version of `esri-loader`, just declare it as a dependency in your own project's `package.json` file. when npm/yarn resolve all your dependencies. you _should_ end up with exactly what you'd expect.

**Comment by vvscode** [comment]
I've tried ) but (at least yarn) installs 2 versions, for react-arcgist newer one

lately I faced an issue, when after removing `yarn.lock` I go

```also all this icons were selectable with inspector now they are not as whole thing is canvas so basically no CSS we are using is applied```

So I'm looking for a way to avoid this in future

**Comment by jgravois** [comment]
works for me. i just released [`v4.0.0`](https://github.com/Esri/react-arcgis/releases/tag/v4.0.0) (in the [@esri](https://www.npmjs.com/package/@esri/react-arcgis) npm namespace) with rev-locked dependencies.

```shell
$ npm view @esri/react-arcgis

@esri/react-arcgis@4.0.0 | Apache-2.0 | deps: 4 | versions: 1
React component library for the ArcGIS API for JavaScript
https://github.com/nicksenger/react-arcgis#readme

keywords: Esri, ArcGIS, TypeScript, React, Map

dist
.tarball: https://registry.npmjs.org/@esri/react-arcgis/-/react-arcgis-4.0.0.tgz
.shasum: afc58a33329c12bc7b363d7767f560ac83f09100
.integrity: sha512-Svb/CDOEnqSEcDmN88uAxbJ664l/pLfJ0HOlQxbTpe3dFrCN9Ey+eBgBFdZojppKEDQjZYiS3Ok5soIyBozqyQ==
.unpackedSize: 109.2 kB

dependencies:
@types/arcgis-js-api: 4.10.0 esri-loader: 2.6.0           react-dom: ^16.5.2           react: ^16.5.2
```
that _should_ do the trick for both `yarn` and `npm` users.
